src/main/java/frc/robot/Robot.java

```diff
@@ -141,7 +141,7 @@ private void updateSmartDashboard() {
 
     SmartDashboard.putBoolean("Has Vision", result.HasResult);
     if (result.HasResult) {
-      mRobotLogger.info(Double.toString(result.distance));
+      // mRobotLogger.info(Double.toString(result.distance));
```

Comment:
Delete commented code, unless you need it for debugging

Reply:
Fixed